Packs/Gmail/Integrations/Gmail/Gmail.py

```diff
@@ -2185,6 +2251,16 @@ def send_as_add_command():
     )
 
 
+def parse_date_isoformat_server(dt: str) -> datetime:
```

Comment:
This is the same as `get_date_isoformat_server` function, no?

Reply:
No, parse_date_isoformat_server gets a string and returns a datetime in a specific format and time zone (used to set lastRun obj) and get_date_isoformat_server gets datetime and returns a parsed string (used to create an incident)

Reply:
Let's add tests for both